### PR TITLE
Add print statement for the console debugging in run_tests.py

### PR DESCRIPTION
**Pull request by samvarankashyap**
$subject

### CODE REVIEW

**Review by jsztuka — APPROVED** [review]
lgtm

File: Dockerfiles/midstream/run_tests.py [hunk 1]
```diff
@@ -68,13 +68,17 @@ def test_for_extract_and_validate_bundle_image(self):
             exit_code = 103
             return exit_code
         result = self.run_extract_operator_bundle()
+        print(result)
+        print(result.stdout)
```
Comment (lslebodn) [inline Dockerfiles/midstream/run_tests.py:72]: It should be logged mainly into output/validation-output.txt

File: Dockerfiles/midstream/run_tests.py [hunk 2]
```diff
@@ -68,13 +68,17 @@ def test_for_extract_and_validate_bundle_image(self):
             exit_code = 103
             return exit_code
         result = self.run_extract_operator_bundle()
+        print(result)
```
Comment (lslebodn) [inline Dockerfiles/midstream/run_tests.py:71]: We should not log internal data structures.

File: Dockerfiles/midstream/run_tests.py [hunk 3]
```diff
@@ -68,13 +68,17 @@ def test_for_extract_and_validate_bundle_image(self):
...
+        print(result.stdout)
         if (result.returncode != 0):
             print("Ansible playbook extract-operator-bundle.yml failed with result code: %s , see the file .errormessage for more info." % result.returncode)
             with open(".errormessage", "w") as error_msg:
                 print("Result code: " + str(result.returncode), "Error message: " + str(result.stderr), file=error_msg)
             exit_code = 50
             return exit_code
         result = self.run_validate_operator_bundle()
+        print(result)
+        print(result.stdout)
```
Comment (lslebodn) [inline Dockerfiles/midstream/run_tests.py:81]: the same as above

**Review by lslebodn — CHANGES_REQUESTED** [review]
-1

**Comment by lslebodn** [comment]
@samvarankashyap bump

**Comment by lslebodn** [comment]
@samvarankashyap  bump ^^

**Comment by samvarankashyap** [comment]
Yes.

**Pull request closed without merging**